src/libANGLE/Context.cpp

```diff
@@ -1870,7 +1870,10 @@ void Context::getIntegervImpl(GLenum pname, GLint *params)
             *params = mState.mCaps.maxLights;
             break;
         case GL_MAX_CLIP_PLANES:
```

Comment:
Why not `GL_MAX_CLIP_DISTANCES_APPLE`?

Reply:
`GL_MAX_CLIP_DISTANCES_APPLE` has the same value as `GL_MAX_CLIP_PLANES`. Therefore I could not create separate case blocks for them.

Reply:
Let's put a `static_assert` for that then, both for future-proofing and for documentation purposes.